### ISSUE TITLE
How to unload driver

### ISSUE THREAD

**Issue by Grazyyt** [z175/kdmapper#3]
how do I unload the driver ?

**Comment by ghost** [comment]
cringe alert, zero the driver and change the return address on the stack to go back to usermode. 

this can be done by calculating the stack size and "deallocating" an entire stack frame so to speak. 
Keep in mind this only zeros the driver, doesn't "free the driver". need a 2 hop rop for that.

stinker.

**Comment by BerkanYildiz** [comment]
ExFreePool ftw

**Comment by ghost** [comment]
yep. but if you can zero the memory first and then rop back to ExFreePool(WtihTag).

**Comment by BigCatGit** [comment]
BSOD appears after setting DriverUnload Router
